gatsby/src/pages/schedule.js

```diff
@@ -3,14 +3,31 @@ import { graphql } from 'gatsby';
 import styled from 'styled-components';
 import Layout from '../components/layout/Layout';
 import SEO from '../components/Seo';
+import Sponsors from '../components/ConferenceSponsors.js';
 
 const PageStyles = styled.div`
-  @media (min-width: 980px) {
-    .padding {
-      padding-top: 40px;
+  @media (max-width: 600px) {
+    .headings {
+      display: block !important;
+      text-align: center;
+    }
+    .sponsors-column {
+      transform-origin: center top !important;
     }
   }
-
+  .headings {
+    display: flex;
+    padding-top: 40px;
+    justify-content: space-between;
+  }
+  h1 {
+    padding: 0 40px 0 0;
+    margin: 22px 0 0 0;
+  }
+  .sponsors-column {
+    transform: scale(0.75);
+    transform-origin: right top;
+  }
```

Comment:
CSS for sponsors on schedule: show in a right column (except on mobile show underneath). Shrink content a bit so logos are not so huge.